docs/source/2020CAPE.rst

```diff
@@ -95,6 +119,22 @@ Synoptic Dynamics
 
     * GFS.v16.0.10 simulates 500hPa HGT fields and MSLP well.
 
+  .. group-tab:: SRW.v1.0
+
+    .. figure:: images/2020CAPE/MSLP_SRW_v1.0_2020CAPE_trim.png
+      :width: 1200
+      :align: center
+
+      Mean sea level pressure (hPa)
+
+    .. figure:: images/2020CAPE/500mb_SRW_v1.0_2020CAPE_trim.png
+      :width: 1200
+      :align: center
+
+      500 hPa geopotential heights (dam) and absolute vorticity (10 :sup:`-5`/s)
```

Comment:
What does the "(dam)" mean ?

Reply:
@linlin-pan can you please answer this question?

Reply:
dam is a height unit which equals 10m.
@climbfuji